script.js

```diff
@@ -0,0 +1,56 @@
+class Library {
+  constructor(books) {
+    this.books = books;
+    this.store = window.localStorage;
+    this.list = document.getElementById('booklist');
+    this.form = document.getElementById('form');
+  }
+
+  displayBooks() {
+    while (this.list.firstChild) {
+      this.list.removeChild(this.list.firstChild);
+    }
+    this.books = JSON.parse(localStorage.getItem('storeBook'));
+    this.books.forEach((book) => {
+      const li = document.createElement('li');
+      li.innerHTML = `author: ${book.author}: title: ${book.title}`;
+      const removeButton = document.createElement('button');
+      removeButton.innerHTML = 'Remove';
+      removeButton.addEventListener('click', () => {
+        this.removeBook(book.title);
+        window.location.reload();
+      });
+      li.appendChild(removeButton);
+      this.list.appendChild(li);
+    });
+  }
+
+  addBook(title, author) {
+    this.books.push({ title, author });
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use array methods such as `concat` over `push` that doesn't mutate the array.